src/ess/reflectometry/types.py

```diff
@@ -93,8 +68,8 @@ class IofQ(sciline.Scope[Run, sc.DataArray], sc.DataArray):
 QBins = NewType('QBins', sc.Variable)
 '''Bins for the momentum transfer histogram.'''
 
-WavelengthEdges = NewType('WavelengthEdges', sc.Variable)
-'''Include only events within the specified edges.'''
+WBins = NewType('WBins', sc.Variable)
+'''Bins for the wavelength histogram'''
```

Comment:
```suggestion
WavelengthBins = NewType('WavelengthBins', sc.Variable)
'''Bins for the wavelength histogram'''
```

see https://scipp.github.io/essreduce/user-guide/reduction-workflow-guidelines.html#c-1-use-common-names-and-types